acra/configuring-maintaining/logging/error_codes.md

```diff
@@ -0,0 +1,11 @@
+---
+weight: 2
+title: Logging error codes
+bookCollapseSection: true
+---
+
+# Logging error codes
+
+Most errors logged by Acra have their own error code. You can find the table of error codes in [logging/event_codes.go](https://github.com/cossacklabs/acra-Q12021/blob/master/logging/event_codes.go).
+
+We plan to use unique event codes for every system event, not only for errors, but these updates are not ready yet. Currently, the default event code is `100` for all events (except errors).
```

Comment:
Speaking of laziness... If this is intended as a reference, this page should include an overview of error codes. At least the most common and stable ones. You can tell people to read the source code for all the gritty details, but if I'm on a page titled "error codes", I'd expect to see at least one error code.